api/src/main/java/org/openmrs/module/kenyaemr/reporting/data/converter/definition/evaluator/ipt/PhysicalAddressDataEvaluator.java

```diff
@@ -35,8 +35,7 @@ public class PhysicalAddressDataEvaluator implements PersonDataEvaluator {
     public EvaluatedPersonData evaluate(PersonDataDefinition definition, EvaluationContext context) throws EvaluationException {
         EvaluatedPersonData c = new EvaluatedPersonData(definition, context);
 
-        String qry = "select init.patient_id,a.land_mark as physical_address from kenyaemr_etl.etl_ipt_initiation init left outer  join kenyaemr_etl.etl_person_address a\n" +
-                "                                                             on init.patient_id = a.patient_id where init.voided = 0;";
+        String qry = "select patient_id,land_mark as physical_address from kenyaemr_etl.etl_person_address;";
```

Comment:
The PersonAddressCalculation is more comprehensive in capturing the physical location than just picking landmark.